PYME/LMVis/visCore.py

```diff
@@ -426,8 +426,51 @@ def add_layer(self, layer):
         self.glCanvas.refresh()
     
         self.layer_added.send(self)
-
     
+    def generate_test_object_layer(self, obj_type):
```

Comment:
As mentioned, not super keen on having test code cluttering this file which is already pretty unwieldy.

Reply:
I think I wrote this before we had that conversation. Since this is not critical to...anything, I will probably just delete the ability to add test objects.